migration/sql-files/062-workitem-id-unique-per-space.sql

```diff
@@ -0,0 +1,91 @@
+-- first, we ADD a new COLUMN for the 'natural id' with the same values as the 'id'
+ALTER TABLE work_items ADD COLUMN "number" integer;
+UPDATE work_items SET number = id;
```

Comment:
Will this update be unique per space? Currently this is using the global ID of a work item as it is of today, right?